### DISCUSSION TITLE
Search Tool (disable enter)

### DISCUSSION THREAD

**Discussion by ghost** [audacious-media-player/audacious#1470] [Q&A] [answered]
Is there a way to disable 'enter' annihilating the current playlist and replacing it with whatever might have partially matched. Nothing kills a groove faster than the Audacious Search...sigh...

**Comment by jlindgren90** [comment]
Create a new playlist called something other than "Now Playing", then it will not be overwritten. The "Now Playing" list is meant to be temporary and can also be replaced by other one-keystroke actions, like opening files from a file manager.

**Reply by ghost** [reply, marked as answer]
This is about as helpful as saying hey spin around three times and do 40 jumping jacks for every new track you want to add. Given the question was can you disable the behavior "No." would have been more inline.

**Reply by jlindgren90** [reply]
Sorry you didn't find my answer helpful, I will make a note not to answer any other questions you may have.

**Comment by ghost** [comment]
You explained the behavior (and provided a bodge workaround) you didn't answer the question.

**Reply by jlindgren90** [reply]
The question was "is there a way". I told you a way (rename the playlist), you just didn't like it.
